sdf_timing/sdfyacc.py

```diff
@@ -380,21 +380,35 @@ def p_del(p):
     p[0] = p[1]
 
 
-def p_device(p):
-    'device : LPAR DEVICE port_spec real_triple real_triple RPAR'
+def p_delval_list(p):
+    '''delval_list : real_triple
+                   | real_triple real_triple
+                   | real_triple real_triple real_triple'''
+
+    # TODO: There should be more possibilities up to 12 real_triples
+    # But we stick with 1, 2 or 3 for now.
```

Comment:
Why are you sticking to 1/2/3 for now?

Reply:
The SDF can express separate timings from transitions between different logic state sets. For now we do not have use for eg. different 0->1 and 1->0 delays. Thats' why I limited parsing to only that triple count that we need.

Reply:
Please add the following info to the comment.

Reply:
Added.